vars/helmExecute.groovy

```diff
@@ -0,0 +1,9 @@
+import groovy.transform.Field
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = "metadata/helmExecute.yaml"
+
+void call(Map parameters = [:]) {
+    List credentials = []
```

Comment:
please add the respective Jenkins credentials, similar to https://github.com/SAP/jenkins-library/blob/master/vars/kanikoExecute.groovy#L7